frontend/public/module/k8s/k8s-ref.ts

```diff
@@ -1,7 +1,6 @@
 // TODO(alecmerdler): Replace all manual string building with this function
-export const referenceForGroupVersionKind = (group: string) => (version: string) => (
-  kind: string,
-) => [group, version, kind].join('~');
+export const referenceForGroupVersionKind = (group: string, version: string, kind: string) =>
+  [group, version, kind].join('~');
```

Comment:
```suggestion
export const referenceForGroupVersionKind = (group: string, version: string, kind: string) =>
  getReference({ group, version, kind });
```
I think we just want to use `getReference`... not sure you even need to rework this call but I'm not against removing that weird `()()()` call structure.